artemis-server/src/main/java/org/apache/activemq/artemis/core/config/impl/ConfigurationImpl.java

```diff
@@ -2852,13 +3038,14 @@ public Configuration setSuppressSessionNotifications(boolean suppressSessionNoti
    }
 
    @Override
-   public String getStatus() {
-      return status;
+   public synchronized String getStatus() {
```

Comment:
Why is status volatile if its use is synchronized?

Reply:
yep, not needed.. thanks

Reply:
sorted